### DISCUSSION TITLE
Installed, and having issues with last step, "Your message cannot be delivered"

### DISCUSSION THREAD

**Discussion by ForceConstant** [mishl-dev/Discord-AI-Chatbot#16] [Q&A]
So it is all very detailed, until the last step "Final step. Invite your bot and DM (Direct message) it or if you want it in channel use !addchannel" So I was able to invite to my server using https://discordjs.guide/preparations/adding-your-bot-to-servers.html#creating-and-using-your-invite-link But when I try to message bot in my server I get "Your message could not be delivered" What else do I need to do?

**Comment by mishl-dev** [comment]
Do you have DM enabled ?

**Comment by mishl-dev** [comment]
Use !addchannel in server of you channel

**Reply by ForceConstant** [reply]
> Use !addchannel in server of you channel

For this I get "discord.ext.commands.errors.CommandNotFound: Command "addchannel" is not found" in the bot logs.

**Reply by mishl-dev** [reply]
> For this I get "discord.ext.commands.errors.CommandNotFound: Command "addchannel" is not found" in the bot logs.

Update your cloned project
